src/api/comment.ts

```diff
@@ -0,0 +1,20 @@
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+export const getComments = (postId?: number) => {
+  return client.get<Comment[]>(`/comments?postId=${postId}`);
```

Comment:
The postId parameter in getComments function is optional. If it is not provided, the generated URL will end with '=' which could lead to unexpected results. Consider adding a check to ensure postId is defined before adding it to the URL.